packages/x-components/src/x-modules/search/components/search-items-injection.mixin.ts

```diff
@@ -19,12 +19,15 @@ export default class SearchItemsInjectionMixin extends Vue {
   /**
    * The computed search items of the entity that uses the mixin.
    *
-   * @remarks It should be defined in the component that uses the mixin and it's intended to be
+   * @remarks It should be overridden in the component that uses the mixin and it's intended to be
    * filled with items from the state. Vue doesn't allow mixins as abstract classes.
-   *
+   * @returns An empty array as fallback in case it is not overridden.
    * @internal
    */
-  protected items!: SearchItem[];
+  @XProvide(SEARCH_ITEMS_KEY)
+  public get items(): SearchItem[] {
```

Comment:
Now TS also complains if you are overriding a property with an accessor. The trick is to define the property here. The bonus part is that we can use the `XProvide` decorator here, and we don't have to use it in every component that uses this mixin.